eclipse-temurin/Dockerfile

```diff
@@ -0,0 +1,86 @@
+#
+# Scala and sbt Dockerfile
+#
+# https://github.com/hseeberger/scala-sbt
+#
+
+# Pull base image
+ARG BASE_IMAGE_TAG
+FROM eclipse-temurin:${BASE_IMAGE_TAG:-11.0.13_8-jdk}
```

Comment:
If this is a 100% copy from the debian docker file we could template the whole FROM instead of the only the TAG?

Reply:
There is actually only 1 difference which is on line 52, the rest would be equal.

We could consolidate them, but I assume for line 52 it requires an `if` for the symlink, in case the file `/opt/java/openjdk/bin/java` exists (to differentiate if its eclipse-temurin)?